Automattic/wp-calypso
# Signup: Change default theme to pub/modern-business

## Changes proposed in this Pull Request

Here we're changing the default theme for the site preview and site style step to `pub/modern-business`.

## Testing instructions

Check that the default theme for the site preview is
http://calypso.localhost:3000/start/onboarding-dev/site-topic-with-preview

<img width="870" alt="Screen Shot 2019-04-23 at 5 11 06 pm" src="https://user-images.githubusercontent.com/6458278/56561560-ceee7f00-65ea-11e9-808a-0343c00f7048.png">

Check that the default selected site style is **'Modern'** 
http://calypso.localhost:3000/start/onboarding-dev/site-style-with-preview

<img width="600" alt="Screen Shot 2019-04-23 at 5 09 41 pm" src="https://user-images.githubusercontent.com/6458278/56561570-d1e96f80-65ea-11e9-8521-3483d05e77ef.png">
